contrib/scripts/cubrid.csh

```diff
@@ -24,54 +24,19 @@ else
 	setenv LD_LIBRARY_PATH $CUBRID/lib:$CUBRID/cci/lib
 endif
 
-set path=($CUBRID/bin $path)
+set path=($CUBRID/bin /usr/sbin $path)
+set curses_lib=
 
 setenv SHLIB_PATH $LD_LIBRARY_PATH
 setenv LIBPATH $LD_LIBRARY_PATH
 
-set LIB=$CUBRID/lib
-
-if ( -f /etc/redhat-release ) then
-        set OS=`cat /etc/system-release-cpe | cut -d':' -f'3-3'`
-else if ( -f /etc/os-release ) then
-        set OS=`cat /etc/os-release | egrep "^ID=" | cut -d'=' -f2-2`
-     endif
-endif
-
-switch ($OS)
-  case "fedoraproject":
-  case "centos":
-  case "redhat":
-  case "rocky":
-  case "oracle":
-    if ( ! -f /lib64/libncurses.so.5 && ! -f $LIB/libncurses.so.5 ) then
-    	ln -s /lib64/libncurses.so.6 $LIB/libncurses.so.5
-    	ln -s /lib64/libform.so.6 $LIB/libform.so.5
-    	ln -s /lib64/libtinfo.so.6 $LIB/libtinfo.so.5
-    endif
-    breaksw
-  case "prolinux":
-    if ( ! -f /usr/lib64/libncurses.so.5 && ! -f $LIB/libncurses.so.5 ) then
-        ln -s /usr/lib64/libncurses.so.6 $LIB/libncurses.so.5
-        ln -s /usr/lib64/libform.so.6 $LIB/libform.so.5
-        ln -s /usr/lib64/libtinfo.so.6 $LIB/libtinfo.so.5
+set is_ncurses5=`ldconfig -p | grep libncurses.so.5 | wc -l`
+if ( $is_ncurses5 == 0 && ! -f $CUBRID/lib/libncurses.so.5 ) then
+  foreach lib ( libncurses libform libtinfo )
+    set curses_lib=`ldconfig -p | grep $lib.so | grep -v "so.[1-4]" | sort -h | tail -1 | awk '{print $4}'`
```

Comment:
I think you shouldn't create Symbolic Links if the curses_lib string is 0.

Reply:
Thank you for your comment!
My mistake while converting script in csh version.